### PR TITLE
profile caching, second pull request changing -> master instead of imthemap

### PR DESCRIPTION
**Pull request by quartzjer**
this replaces the pull request #548 to go into master now (couldn't figure out how to change the original one)

### CODE REVIEW

File: Common/node/synclet/dataaccess.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ module.exports = function(app) {
         });
     });
 
+    // depreciated 2012-01-12 by jer, pretty sure nothing uses this
```
Comment (othiym23) [inline Common/node/synclet/dataaccess.js]: Pretty sure you mean "deprecated" here, boss.

Comment (othiym23) [inline Common/node/synclet/dataaccess.js]: Unless there's an amortization schedule on our code base I don't know about, that is.

**Comment by othiym23** [comment]
This has been working great for me, I like the idea, and I've already added support for it to my two connector branches, so I am solidly +1.

**Comment by othiym23** [comment]
Also, I need this feature to do Singly's account settings properly, so double +1.

**Comment by quartzjer** [comment]
spel fix, merging :)

**Pull request merged**